frontend/tests/controllers/CertificatesTest.php

```diff
@@ -54,25 +54,21 @@ public function testGenerateContestCertificates() {
      */
     public function testConnectionRabbitMQ() {
         //Create user
-        ['identity' => $identity] = \OmegaUp\Test\Factories\User::createUser();
+        ['identity' => $certificateGenerator] = \OmegaUp\Test\Factories\User::createUser();
 
         //Create users
-        ['identity' => $test1] = \OmegaUp\Test\Factories\User::createUser();
-        ['identity' => $test2] = \OmegaUp\Test\Factories\User::createUser();
-        ['identity' => $test3] = \OmegaUp\Test\Factories\User::createUser();
-        ['identity' => $test4] = \OmegaUp\Test\Factories\User::createUser();
-        ['identity' => $test5] = \OmegaUp\Test\Factories\User::createUser();
-        ['identity' => $test6] = \OmegaUp\Test\Factories\User::createUser();
-        ['identity' => $test7] = \OmegaUp\Test\Factories\User::createUser();
-        ['identity' => $test8] = \OmegaUp\Test\Factories\User::createUser();
+        $identities = [];
+        for ($i = 0; $i < 8; $i++) {
```

Comment:
Acá podemos guardar ese número de identidades a crear en una variable?

De esa forma ya podríamos utilizar algo como esto:
```suggestion
        foreach (range(0, $numOfIdentities - 1) as $index) {
```
que es un poco más entendible.